cmd/loom/gateway/root_cmd.go

```diff
@@ -28,6 +30,8 @@ func newRootCommand() *cobra.Command {
 	pflags.StringVarP(&gatewayCmdFlags.EthPrivKeyPath, "eth-key", "", "", "Ethereum Private Key file path")
 	pflags.StringVarP(&gatewayCmdFlags.HSMConfigPath, "hsm", "", "", "HSM file path")
 	pflags.StringVarP(&gatewayCmdFlags.Algo, "algo", "", "ed25519", "Signing algorithm")
+	pflags.StringVarP(&gatewayCmdFlags.FormatRaw, "raw", "", "", "Raw output format")
+	pflags.StringVarP(&gatewayCmdFlags.FormatJSON, "json", "", "", "JSON output format")
```

Comment:
Don't add the new flags here, add them in `newListContractMappingsCommand()`.
And use `BoolVar`, not `StringVar`